<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacketProcessor</class>
 <widget class="QMainWindow" name="PacketProcessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 217, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="PlotWidget" name="graphicsView" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>41</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="path_label">
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>41</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="changeDir_btn">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border:none;
	border-radius: 8px;
}

QPushButton:hover {
	background-color: rgb(199, 205, 255);
}</string>
            </property>
            <property name="text">
             <string>Выбрать папку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="process_btn">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border:none;
	border-radius: 8px;
}

QPushButton:hover {
	background-color: rgb(199, 205, 255);
}</string>
            </property>
            <property name="text">
             <string>Обработать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
